e2e/claims.service.e2e.ts

```diff
@@ -507,6 +544,23 @@ describe('Enrollment claim tests', () => {
       ).rejects.toEqual(new Error(ERROR_MESSAGES.ROLE_PREREQUISITES_NOT_MET));
     });
 
+    test.only('enrollment with credential/claim time expiration', async () => {
```

Comment:
Is the `.only` correct?
```suggestion
    test('enrolment with credential/claim time expiration', async () => {
```

Reply:
I removed it here https://github.com/energywebfoundation/iam-client-lib/pull/583